tests/steps/application_v2_steps.py

```diff
@@ -344,14 +419,19 @@ def remember_app_id(context):
     context.app_ids.append(app_id)
 
 
-def wait_for_algod_transaction_processing_to_complete():
+@then(
+    "I sleep for {millisecond_num} milliseconds for indexer to digest things down."
+)
+def wait_for_algod_transaction_processing_to_complete(
```

Comment:
Is this being used as both a cucumber step function as well as called directly as a helper function by other steps? If so, I recommend refactoring so that the helper function remains as such (not a step), while the new step immediately calls the helper. Optionally, consider moving the helper near the top of the file.

Reply:
I think this suggestion makes sense, and I will refactor this part with an updated comment.

Reply:
Addressed in https://github.com/algorand/py-algorand-sdk/pull/348/commits/35a1c43bc844a8fdbb7817e1252b2c405f40e102 - let me know how this looks